pgmpy/tests/test_models/test_FactorGraph.py

```diff
@@ -280,5 +280,25 @@ def test_check_model4(self):
         self.graph.add_edges_from([('a', phi3), ('c', phi3)])
         self.assertRaises(ValueError, self.graph.check_model)
 
+    def test_copy(self):
+        self.graph.add_nodes_from(['a', 'b', 'c'])
+        phi1 = DiscreteFactor(['a', 'b'], [2, 2], np.random.rand(4))
+        phi2 = DiscreteFactor(['b', 'c'], [2, 2], np.random.rand(4))
```

Comment:
@lohani2280 I am seeing these 2 statements repeated elsewhere in this class. Can you please move all of them to the `setUp` method. Also make sure to put that in a separate commit.